Source/Scene/Cesium3DTileset.js

```diff
@@ -898,6 +899,14 @@ function Cesium3DTileset(options) {
    */
   this.debugShowUrl = defaultValue(options.debugShowUrl, false);
 
+  /**
+   * The primitive to be rendered during the pick pass instead of the tileset.
+   *
+   * @type {Primitive}
+   * @private
```

Comment:
I don't think we ever really decided if `pickPrimitive` should be private or not, but I'm leaning towards making it public because it allows anyone to create pick proxies for their tilesets. Can you remove `@private` from here?